test/exactly.test.js

```diff
@@ -20,7 +20,7 @@ class Fixture extends React.Component {
 
 const it = createTest(<Fixture />)
 
-describe('#exactly', () => {
+describe.only('#exactly', () => {
```

Comment:
Please revert this - also, you can use `--grep` in testing and you'll never need to manually mark a test as .only again :-)